src/Ethereum/ABI/Parameters.cpp

```diff
@@ -59,7 +59,11 @@ std::string ParamSet::getType() const {
     std::string t = "(";
     int cnt = 0;
     for (auto p : _params) {
-        if (cnt++ > 0) {
+        bool isFirst = cnt++ > 0;
+        if (p->shouldSkipEncodeInFunctionSignature) {
```

Comment:
We should look to submit this P/R upstream to get feedback from the trustwallet maintainers - I do not understand this enough to know if altering the Ethereum ABI is safe to do